hack/update-bazel.sh

```diff
@@ -43,3 +44,6 @@ fi
 set -o xtrace
 "$gazelle" fix --external=external
 "$kazel" --cfg-path=./.kazelcfg.json
+find . -name BUILD -o -name BUILD.bazel -o -name '*.bzl' -type f \
+  \( -not -path '*/vendor/*' -prune \) \
```

Comment:
should we also buildifier BUILD files in vendor?

Reply:
I don't think so? 

Either we're using the files downloaded from the repo -- in which case we shouldn't futz with the formatting) -- or we used gazelle to generate them.

Reply:
SGTM :+1: